app/domain/output_document.rb

```diff
@@ -101,4 +101,10 @@ def template_for(section)
     File.read(
       Rails.root.join('app', 'templates', "output_document_#{section}.html.erb"))
   end
+
+  def to_currency(number_string)
+    return '' if number_string.blank?
+
+    number_to_currency(number_string, precision: 0)
```

Comment:
`number_to_currency` takes `number` as an argument, `number_string` is potentially misleading.